src/translations/en.json

```diff
@@ -272,6 +272,7 @@
         "mustTargetPointCard": "You can only scuttle a card your opponent has played for points"
       },
       "oneOffs": {
+        "wrongPhase": "You can't resolve a five right now.",
```

Comment:
This belongs in the inner "five" block on line 287 because it is an error specific to the five one-off. Please also add the spanish and french translations in the same places in es.json and fr.json:
```
// es.json
"wrongPhase": "No puedes resolver un cinco en este momento.",
```

```
// fr.json
"wrongPhase": "Vous ne pouvez pas résoudre un cinq maintenant.",
```